app/views/rb_burndown_charts/show.html.erb

```diff
@@ -43,8 +43,8 @@ See doc/COPYRIGHT.rdoc for more details.
 
   <div class="burndown_chart" id="burndown_<%= @burndown.sprint_id %>" style="width:600px;height:300px;"><div class="loading"><%=l('backlogs.generating_chart')%></div></div>
 
-  <fieldset class="burndown_control">
-      <legend><%= l('backlogs.chart_options') %></legend>
+  <fieldset class="burndown_control form--fieldset">
```

Comment:
@vcostin it looks like `burndown_control` also applies some styling. Can we remove this?